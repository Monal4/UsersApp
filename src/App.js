import './App.css';
import Routes from './util/routes';
import React from 'react';

const App = () => {
  return (
      <main>
        <Routes />
      </main>
  );
}

export default App;
