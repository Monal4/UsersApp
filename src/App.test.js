import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

import {Provider} from 'react-redux';
import store from './redux-files/store'

test('renders learn react link', () => {
    const component = renderer.create(
        <Provider store={store}>
            <App/>
        </Provider>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
