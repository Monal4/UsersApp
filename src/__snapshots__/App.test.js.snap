// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<main>
  <div>
    <div
      className="tableDiv"
    >
      <div
        className="searchBox"
      >
        <input
          onChange={[Function]}
          placeholder="search Users"
          type="search"
        />
      </div>
      <table
        className="table"
      >
        <thead
          className="thead"
        >
          <tr>
            <th>
               
              Name
               
            </th>
            <th>
               
              Email
               
            </th>
            <th>
               
              City
               
            </th>
            <th>
               
              Company
               
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
  </div>
</main>
`;
